workflows/smc.py

```diff
@@ -36,10 +36,10 @@ def write_smcpp_file(path, mask_file, selection=None):
         cmd = f"tabix -p bed {inter_mask_path}"
         logging.info("Running:" + cmd)
         subprocess.run(cmd, shell=True, check=True)
-        cmd = f"docker run --rm -v $PWD:/mnt terhorst/smcpp:latest vcf2smc --mask {inter_mask_path} {vz_file} "
+        cmd = f"smc++ vcf2smc --mask {inter_mask_path} {vz_file} "
         cmd = cmd + f"{smc_file} {chr_name} pop1:"
     else:
-        cmd = f"docker run --rm -v $PWD:/mnt terhorst/smcpp:latest vcf2smc {vz_file} {smc_file} {chr_name} pop1:"
+        cmd = f"smc++ vcf2smc {vz_file} {smc_file} {chr_name} pop1:"
```

Comment:
these commits to `smc.py` came over from #44. we still need to decide once and for all if we will include `smc++` so i've decided to keep these changes here.